assets/js/components/SaptuneDetails/SaptuneDetails.stories.jsx

```diff
@@ -119,3 +119,37 @@ export const StagingDisabled = {
     staging: { enabled: false, notes: [], solutions_ids: [] },
   },
 };
+export const SaptuneServiceStatusPassing = {
+  args: {
+    ...Default.args,
+
+    services: [
+      { name: 'saptune', enabled: 'enabled', active: 'active' },
+      { name: 'sapconf', enabled: 'disabled', active: 'inactive' },
+      { name: 'tuned', enabled: 'disabled', active: 'inactive' },
+    ],
+  },
+};
+export const SaptuneServiceStatusFailing = {
+  args: {
+    ...Default.args,
+
+    services: [
+      { name: 'saptune', enabled: 'disabled', active: 'inactive' },
+      { name: 'sapconf', enabled: 'disabled', active: 'active' },
+      { name: 'tuned', enabled: null, active: null },
```

Comment:
Is `null/null` failing?

Reply:
No there is no failing state for tuned, i thought better nothing then something in the failing case :D 

`  tuned: {
    'enabled/active': <HealthIcon health="warning" />,
    'enabled/inactive': <HealthIcon health="warning" />,
    'disabled/active': <HealthIcon health="warning" />,
    'disabled/inactive': <HealthIcon health="passing" />,
  },`
  
  Or what do you think? i also can delete it :D

Reply:
Okey, it is fine them